encoding/descaling.md

```diff
@@ -0,0 +1,356 @@
+# Descaling
+
+The ability to descale is a wonderful tool
+to have in any encoder's arsenal.
+You may be familiar with
+how most anime are not native 1080p,
+but a lower resolution.
+But how do we make use of that?
+How do you find the native resolution
+and reverse the upscale?
+
+
+## When and where to descale
+
+There's a lot of opportunities
+where descaling might prove beneficial.
+For example say you've got
+a very blurry Blu-ray source.
+Rather than sharpening it,
+you might want to consider
+checking if it's possible to descale it
+and maybe alleviate a lot of
+the blurriness that way.
+Or the opposite:
+say you've got a source full of ringing.
+It might've been upscaled
+using a very sharp kernel,
+so it's worth a try
+to see if it can be descaled.
+It's no surprise that descaling tends to
+offer far better lineart
+than usual resampling does.
+
+However, descaling is not always an option.
+The worse your source is,
+the less likely it is that
+descaling will be a better alternative
+to a simple resample.
+If you've got a source
+full of broken gradients
+or noise patterns,
+like your average simulcast stream,
+descaling might only hurt the overall quality.
+Sadly, sources with a lot of post-processing
+might also prove tough to properly descale
+without dabbling into specific masks.
+However, so long as you've got
+a source with nice, clean lineart,
+descaling might be a viable option,
+and possibly nullify the need
+of a lot of other filtering.
+
+
+## Preparation
+
+To prepare for figuring out the native resolution,
+you'll want to use [getnative][],
+a Python script designed around
+figuring out the resolution
+a show was animated at.
+For the actual descaling,
+make sure to grab BluBb-mADe's [descale][].
+
+One important thing
+to keep in mind when descaling
+is that you will never find
+"the perfect descaling settings".
+This is because the best available sources to consumers,
+usually Blu-rays,
+aren't lossless.
+There's always going to be some differences
+from the original masters
+which makes it impossible
+to perfectly descale something.
+However, usually those differences
+are so small that they're negligible.
+If you run into a case where
+you can only get high relative errors,
```

Comment:
Misses an emphasis on "high relative errors using/given by the script".
It might confuse beginners who have no prior knowledge of descaling.